### PR TITLE
Query a single subscription

### PR DESCRIPTION
**Pull request by pboers1988**

### CODE REVIEW

File: test/unit_tests/graphql/test_subscription.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+from http import HTTPStatus
+from uuid import uuid4
+
+
+def build_simple_query(subscription_id):
+    q = f"""
```
Comment (mrijk) [inline test/unit_tests/graphql/test_subscription.py:19]: I think we can make these queries a lot more readable by using graphql variables (like we do in Pythia)

**Review by mrijk — APPROVED** [review]
LGTM, apart from the difficult to read queries

File: test/unit_tests/graphql/test_subscription.py (hunk 1 above)
Comment (tjeerddie) [reply test/unit_tests/graphql/test_subscription.py:19]: create a issue to update this :3

**Review by tjeerddie — DISMISSED** [review]

File: test/unit_tests/graphql/test_subscription.py (hunk 1 above)
Comment (mrijk) [reply test/unit_tests/graphql/test_subscription.py:19]: Why? This is a small change

**Pull request merged**